docs/API.md

```diff
@@ -526,14 +547,16 @@ const obj = stampit()
   bar: 'bar'
 })
 .compose(concreteStamp, additionalStamp, utilityStamp)
-.create();
+.create(42, 511, 3.14);
```

Comment:
Uhh, how these number are helping to explain anything? Since there is no reference to these in the example, it's just bogus and useless here ;)